packages/godcap/bonsai/client.go

```diff
@@ -169,6 +185,23 @@ func (c *Client) UploadInput(input []byte) (string, error) {
 	return response.Uuid, nil
 }
 
+func (c *Client) UploadImage(imageID string) error {
```

Comment:
should the `imageID` match the content of `elf`? If so, I think it would be better to have the user to provide the `elf` data directly.